solutions/aspnet/kill.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+pid=$(ps -eaf | grep /app/webapp | grep -v grep | awk '{ print $2 }')
+if [ ! -z "${pid}" ]; then
+    sudo kill -9 ${pid}
```

Comment:
The previous approach using `server.pid` ensures the the same `webapp` instance we just launched get killed. This script actually would find the first instance of `webapp` and kill it. What's the reason to switch to the current one?

Reply:
This will actually find all instances and kill them, not just the first - its tighter on its approach and ensures that we are killing things that are actually running, and killing the right process